docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -118,162 +128,126 @@ export default function PlainCssButtonDeep() {
 Explicitly providing the class names to the component is too much effort?
 [You can target the class names generated by Material-UI](/styles/advanced/#with-material-ui-core).
 
-[![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/global-css-bir9e)
+[![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/global-classnames-mc4wl)
 
-**GlobalCssButton.css**
+**GlobalCssSlider.css**
 
 ```css
-.MuiButton-root {
-  background-color: #6772e5;
-  box-shadow: 0 4px 6px rgba(50, 50, 93, 0.11), 0 1px 3px rgba(0, 0, 0, 0.08);
-  padding: 7px 14px;
+.MuiSlider-root {
+  color: #20b2aa;
 }
-.MuiButton-root:hover {
-  background-color: #5469d4;
+.MuiSlider-root:hover {
+  color: #2e8b57;
 }
-.MuiButton-label {
-  color: #fff;
+.MuiSlider-root .MuiSlider-thumb {
+  border-radius: 30%;
 }
 ```
 
 **GlobalCssButton.js**
 
 ```jsx
 import * as React from 'react';
-import Button from '@material-ui/core/Button';
-import './GlobalCssButton.css';
+import Slider from '@material-ui/lab/SliderStyled';
+import './GlobalCssSlider.css';
 
-export default function GlobalCssButton() {
-  return <Button>Customized</Button>;
+export default function GlobalCSSSlider() {
+  return <Slider defaultValue={30} />;
 }
 ```
 
-### Controlling priority ⚠️
-
-**Note:** JSS injects its styles at the bottom of the `<head>`. If you don't want to mark style attributes with **!important**, you need to change [the CSS injection order](/styles/advanced/#css-injection-order), as in the demo:
-
-```jsx
-import { StylesProvider } from '@material-ui/core/styles';
-
-<StylesProvider injectFirst>{/* Your component tree.
-      Now, you can override Material-UI's styles. */}</StylesProvider>;
-```
-
 ## Styled Components
 
 ![stars](https://img.shields.io/github/stars/styled-components/styled-components.svg?style=social&label=Star)
 ![npm](https://img.shields.io/npm/dm/styled-components.svg?)
 
-The `styled()` method works perfectly on all of the components.
+By default the Material-UI components, comes with emotion as their style engine. However,
+if you would like to use `styled-components`, you can configure your app, by following this [example project](https://github.com/mui-org/material-ui/blob/next/examples/create-react-app-with-styled-components/README.md).
```

Comment:
So we can easily jump in the source (see the files & markdown at the same time)

```suggestion
if you would like to use `styled-components`, you can configure your app, by following this [example project](https://github.com/mui-org/material-ui/blob/next/examples/create-react-app-with-styled-components).
```

Reply:
Later on, I think that we will need to have a proper documentation page for the switch between the two engines, once we have something stable.

Reply:
Agree 👍